temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Avoid using print statements for logging or debugging in production code. Consider using the `logging` module for better control over log levels and output formatting.
```
import logging

logging.basicConfig(level=logging.INFO)
logging.info("hello")
```